packages/@aws-cdk/aws-eks/test/nodegroup.test.ts

```diff
@@ -139,6 +140,52 @@ describe('node group', () => {
     });
 
 
+  });
+
+  test('create a bottlerocket nodegroup correctly', () => {
+    // GIVEN
+    const { stack, vpc } = testFixture();
+
+    // WHEN
+    const cluster = new eks.Cluster(stack, 'Cluster', {
+      vpc,
+      defaultCapacity: 0,
+      version: CLUSTER_VERSION,
+    });
+    new eks.Nodegroup(stack, 'Nodegroup', {
+      cluster,
+      amiType: NodegroupAmiType.BOTTLEROCKET_X86_64,
```

Comment:
Nice work !!!
Please also test the `amiType: NodegroupAmiType.BOTTLEROCKET_ARM_64` case, thanks.